libs/shared/methodologies/bold/processors/src/geolocation-precision-processor.ts

```diff
@@ -143,10 +145,11 @@ export abstract class GeolocationPrecisionRuleProcessor extends RuleDataProcesso
       return {};
     }
 
-    const participantHomologationDocument = getParticipantHomologationDocument({
-      homologationDocuments,
-      participantId: massDocumentEvent.participant.id,
-    });
+    const participantHomologationDocument =
+      getParticipantHomologationDocumentById({
```

Comment:
I know this makes the name quite big, but the `ById` looks to me like we need to pass the homologation id, not the participant id
```suggestion
      getParticipantHomologationDocumentByParticipantId({
```

Reply:
[feature/check-participants-homologation 1dcbe60] refactor(rule): update homologation document helper functions and usage